src/systemd/meson.build

```diff
@@ -77,11 +77,14 @@ endif
 foreach header : _systemd_headers + _not_installed_headers + [libudev_h_path]
         foreach opt : opts
                 std_name = opt.length() == 2 ? '_'.join(opt[1].split(':')) : ''
-                name = ''.join(['cc-', header.split('/')[-1], '_', opt[0], std_name])
-                test(name,
-                     check_compilation_sh,
-                     args : cc.cmd_array() + ['-c', '-x'] + opt +
-                             ['-Werror', '-include',
-                             meson.current_source_dir() / header])
+                test(''.join(['cc-', header.split('/')[-1], '_', opt[0], std_name]),
```

Comment:
That join isn't doing much here…
`test('cc-' + header.split('/')[-1] + '_' + opt[0] + std_name, …` ?